examples/slurm-bare-metal-dask-rapids/ansible-prereqs.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Anaconda on all hosts
+- hosts: all
+  become: yes
+  roles:
+  - andrewrothstein.miniconda
+
+# CUDA components just on workers
+- hosts: slurm-workers
+  become: yes
+  roles:
+  - nvidia-ml
+
+  tasks:
```

Comment:
Should we perhaps include a CUDA role or the driver playbook here instead? The driver role checks for GPUs and skips if they don't exist, but in a mixed CPU/GPU cluster this task section fails.

Alternatively, we could include something like this somewhere (from the driver playbook):

```
  tasks:
    - name: install custom facts
      include_role:
        name: facts

    - name: install nvidia driver
      include_role:
        name: nvidia-driver
      when: ansible_local['gpus']['count']
```

Reply:
@dholt: 👍 I'll see if I can refactor this into a CUDA role that does the right checks.